File: src/main/resources/static/src/app/shared/product-selector/negative-end-page.ts
import { JsonMember, JsonObject } from 'typedjson-npm/src/typed-json';
/**
 * Object to represent negativeEndPage of the product selector
 * @author ccasallas
 *
*/
@JsonObject
export class NegativeEndPage {
    @JsonMember
    titleText: string;
    @JsonMember
    goAnywayText: string;
    @JsonMember
    whyNotButtonText: string;
    @JsonMember
    productLink: string;
    @JsonMember
    noThanksText: string;
}

File: src/main/java/co/com/fsighttech/imp/issuing/bo/NegativeEndPage.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo;

import lombok.Getter;
import lombok.Setter;

/**
 * Represents the Page when user has identified that product is not the correct
 * one.
 * 
 * @author jtorres
 *
 */
@Getter
@Setter
public class NegativeEndPage {

	private String titleText;
	private String goAnywayText;
	private String whyNotButtonText;
	private String productLink;
	private String noThanksText;

//	public NegativeEndPage(String titleText, String goAnywayText, String whyNotButtonText, String productLink,
//			String noThanksText) {
//
//		this.titleText 			=  titleText;
//		this.goAnywayText		=  goAnywayText;
//		this.whyNotButtonText	=  whyNotButtonText;
//		this.productLink		=  productLink;
//		this.noThanksText		=  noThanksText;
//	}
}
